src/struct_info.rs

```diff
@@ -126,6 +126,13 @@ impl<'a> StructInfo<'a> {
             quote!(#[doc(hidden)])
         };
 
+        let mut b_decl_generics = b_generics.clone();
+        for param in &mut b_decl_generics.params {
+            if let syn::GenericParam::Type(ref mut param) = param {
```

Comment:
Wouldn't this match **all** type parameters? The one you want to add a default to is always the first one, so you can just modify that.

Or even better - just add the default when creating `b_generics`.

Reply:
Yes, that would match all parameters. I guess that barely make sense.

Though unfortunately rust doesn't allow to specify defaults for non-trailing generics and the `TypedBuilderFields` is the first one. I'm not sure whether it's fine to set it as the last one, perhaps it might break some code.

Reply:
I'm half minded to just go with your original design as a temporary solution. Once const generics support strings I plan on doing a great refactor that'll break anything (other than regular by-the-book usage) anyway, and once that come I may remove it in favor of something else.